applications/notes/HM.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+<body>
+<header>
+    <section>
+        <h1> 
+            Notedly
+
+        </h1>
+    </section>
+    <hr />
+    <h2>
+        <ul>
+            <li> Home</li>
+            <li>My Notes</li>
+            <li>Favourites</li>
+            <li>New</li>
+        </ul>
+    </h2>
```

Comment:
Your `<h2>` here, and the `<h3>` below, shouldn't be used to include entire lists. Think of them more like titles in a book: `<h1>` for the book title, `<h2>` for chapter titles, `<h3>` for section titles, and so on.